lib/ReactViewModels/SearchState.ts

```diff
@@ -14,13 +14,13 @@ import CatalogSearchProvider from "../Models/SearchProviders/CatalogSearchProvid
 
 interface SearchStateOptions {
   terria: Terria;
-  catalogSearchProvider?: any;
-  locationSearchProviders?: any[];
+  catalogSearchProvider?: CatalogSearchProvider;
+  locationSearchProviders?: SearchProvider[];
 }
 
 export default class SearchState {
   @observable
-  catalogSearchProvider: SearchProvider | undefined;
+  catalogSearchProvider: CatalogSearchProvider | undefined;
```

Comment:
Can the type here be the abstract `SearchProvider` so that it leaves the window open for others to write a custom search provider if they want one?

Reply:
Yep done